src/v3/README.md

```diff
@@ -1,12 +1,152 @@
+
+### Browsers
+
+The third generation Sign-In Widget supports most [standard browsers](https://help.okta.com/oie/en-us/Content/Topics/Miscellaneous/Platforms_Browser_OS_Support.htm). It doesn't support these browsers:
+
+- Internet Explorer
+- Embedded browsers like Microsoft Office
+- Hardware integrations like hand-held Android devices
+
+### Deployment
+
+You can use the third generation Sign-In Widget in the following ways:
+
+- Redirect authentication, sign-in page (default): Okta provides a sign-in page that is available at your org's URL. By default, a user who signs in on this page is redirected to the Okta user dashboard.
+- Redirect authentication, sign-in page (customizable): Okta provides a sign-in page that you can customize and make available under a custom domain that is a subdomain of your company's top-level domain.
+
+The third generation Sign-In Widget doesn't support embedded authentication.
+
+See [Okta deployment models - redirect vs. embedded](https://developer.okta.com/docs/concepts/redirect-vs-embedded/).
+
+### Authenticators
+
+The third generation Sign-In Widget doesn't currently support these authenticators:
+
+- Duo Security
+- RSA
+- Symantec VIP
+- IdP authenticator
+- Social login
+- Custom Authenticator
+
+### Branding
+
+The third generation Sign-In Widget makes multibrand customization easier, but it doesn't support custom CSS overrides. If your org uses the code editor to customize a sign-in page, those customizations don't migrate. See [Style the sign-in page](https://developer.okta.com/docs/guides/custom-widget/main/).
+
+## Compare generations
+
+| Feature | Second generation | Third generation |
+| ---                | ---                            | ---             |
+| Deployment | Redirect (Okta-hosted)</br>Embedded (self-hosted)</br>Pinning to a specific Okta-hosted version | Redirect (Okta-hosted)</br>Automatic deployment of latest EA version |
+| Authenticators | Email</br>Password</br>Okta Verify (Okta FastPass, push, TOTP)</br>WebAuthn</br>Smart Card IdP</br>Google Authenticator</br>YubiKey OTP</br>SMS</br>Security Question</br>Custom</br>Duo Security</br>RSA</br>Symantec VIP</br>IdP authenticator</br>Social login | Email</br>Password</br>Okta Verify (Okta FastPass, push, TOTP)</br>WebAuthn</br>Smart Card IdP</br>Google Authenticator</br>YubiKey OTP</br>SMS</br>Security Question |
+| Flows | Profile enrollment</br>Authenticator enrollment</br>Authenticator verification</br>Forgot password</br>Account unlock | Profile enrollment</br>Authenticator enrollment</br>Authenticator verification</br>Forgot password</br>Account unlock |
+| Globalization | Right-to-left languages (requires customization)</br>All out-of-the-box translations</br>String customization</br>Bring your own language | All out-of-the-box translations</br>String customization</br>Bring your own language |
+| Branding | Branding and multibrands</br>CSS overrides (code editor) | Branding and multibrands |
+
+
+## Type of Applications
```

Comment:
@lesterchoi-okta Is this section required in the 3rd gen README? if so, what should we include? I can repurpose content from the devdocs, or even link to the devdocs site.